packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.tsx

```diff
@@ -84,20 +82,6 @@ export const ChoiceCardsBanner = ({
     const [choiceCardSelection, setChoiceCardSelection] = useState<
         ChoiceCardSelection | undefined
     >();
-    const [hasBeenSeen, setNode] = useHasBeenSeen({ threshold: 0 }, true) as HasBeenSeen;
-
-    useEffect(() => {
```

Comment:
the view + insert events are already sent from the BannerWrapper

Reply:
One thing’s confusing me: if the view and insert events being sent from the wrapper means they don’t need to be sent here, why doesn’t it also mean they don’t need to be sent in the ChoiceCards element?

Reply:
The ChoiceCards component sends an additional event with componentType of `ACQUISITIONS_OTHER`.
This is to enable tracking of _amounts_ AB test impressions.
It's separate from the tracking of the banner as a whole, which is what those events from the BannerWrapper are for